common/djangoapps/microsite_configuration/microsite.py

```diff
@@ -66,7 +66,15 @@ def get_value_for_org(org, val_name, default=None):
     This returns a configuration value for a microsite which has an org_filter that matches
     what is passed in
     """
-    return BACKEND.get_value_for_org(org, val_name, default)
```

Comment:
I think removing this from the backend and leaving it here, prevents creating a backend with an extension. e.g: A backend that handles multiple orgs.

This can well go to the base backend to avoid repeting code, but the ability to have an alternative implementation of the method I think is important.